test/e2e/scripts/setup-instance/00-entrypoint-gitlab.sh

```diff
@@ -37,11 +37,11 @@ tee specification.json << EOF
   ],
   "UserData": "${IGNITION_BASE64}",
 
-  "SubnetId": "subnet-c18341ed",
+  "SubnetId": "subnet-05d7c6b1b5cfea811",
   "IamInstanceProfile": {
     "Name": "ci-datadog-agent-e2e-runner"
   },
-  "SecurityGroupIds": ["sg-0f5617ceb3e5a6c39"]
+  "SecurityGroupIds": ["sg-019917348cb0eb7e7"]
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ Could these two re-use the variables set above?